Hyper/PlayerData/InventorySystem/Items/Lamp.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Common.Meshes;
+using OpenTK.Mathematics;
+using Physics.TypingUtils;
+
+namespace Hyper.PlayerData.InventorySystem.Items;
+internal class Lamp : Item
+{
+    public override string Id => "lamp";
+
+    public override bool IsStackable => true;
+
+    public override void Use(Scene scene)
```

Comment:
I think picking up lamps should be under right mouse button so just Secondary use method